site/docs/master/install-overview.md

```diff
@@ -1,46 +1,112 @@
+#### Configure more than one storage location for backups or volume snapshots
 
-Second, if you need to back up persistent volume data, you must select a volume backup solution. [volume snapshot providers][0] contains information on the supported options. For example, if you use [Portworx][6] for persistent storage, you can install their Velero plugin to get native Portworx snapshots as part of your Velero backups. If there is no native snapshot plugin available for your storage platform, you can use Velero's [restic integration][4], which provides a
-platform-agnostic backup solution for volume data.
+Velero supports any number of backup storage locations and volume snapshot locations. For more details, see [about locations](locations.md).
 
-## Customize configuration
+However, `velero install` only supports configuring at most one backup storage location and one volume snapshot location.
 
-Whether you run Velero on a cloud provider or on-premises, if you have more than one volume snapshot location for a given volume provider, you can specify its default location for backups by setting a server flag in your Velero deployment YAML.
+To configure additional locations after running `velero install`, use the `velero backup-location create` and/or `velero snapshot-location create` commands along with provider-specific configuration. Use the `--help` flag on each of these commands for more details.
 
-For details, see the documentation topics for individual cloud providers.
+#### Install an additional volume snapshot provider
 
-## Installing with the Helm chart
+Velero supports using different providers for volume snapshots than for object storage -- for example, you can use AWS S3 for object storage, and Portworx for block volume snapshots.
+
+However, `velero install` only supports configuring a single matching provider for both object storage and volume snapshots.
+
+To use a different volume snapshot provider:
+
+1. Install the Velero server components by following the instructions for your **object storage** provider
+
+1. Add your volume snapshot provider's plugin to Velero (look in [your provider][0]'s documentation for the image name):
+
+    ```bash
+    velero plugin add <PLUGIN-IMAGE>
+    ```
+
+1. Add a volume snapshot location for your provider, following [your provider][0]'s documentation for configuration:
+
+    ```bash
+    velero snapshot-location create <NAME> \
+        --provider <PROVIDER-NAME> \
+        [--config <PROVIDER-CONFIG>]
+    ```
+
+#### Generate YAML only
+
+By default, `velero install` generates and applies a customized set of Kubernetes configuration (YAML) to your cluster.
+
+To generate the YAML without applying it to your cluster, use the `--dry-run -o yaml` flags.
+
+This is useful for applying bespoke customizations, integrating with a GitOps workflow, etc.
+
+#### Providing credentials when installing with the Helm chart
+
+TODO can this go in the Helm chart docs?
```

Comment:
Yes, I think this documentation should go here instead: https://github.com/helm/charts/tree/master/stable/velero#secret-for-cloud-provider-credentials

Reply:
:+1: let's get a PR in to the helm chart that covers this and #1819